package.json

```diff
@@ -12,6 +12,9 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
+        "npm": "2.15.10"
```

Comment:
Fair enough. `@3` would bring some nice benefits, but I do see how testing for it might be unfeasible at this point. I'd like to see this land sooner than the later though, preferably with `3` 😼